draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1251,24 +1251,26 @@ user agent MUST process the cookie-av as follows.
 If the attribute-name case-insensitively matches the string "Max-Age", the
 user agent MUST process the cookie-av as follows.
 
-1.  If the first character of the attribute-value is not a DIGIT or a "-"
-    character, ignore the cookie-av.
+1.  If the attribute-value is empty, ignore the cookie-av.
 
-2.  If the remainder of attribute-value contains a non-DIGIT character, ignore
+2.  If the first character of the attribute-value is neither a DIGIT, nor a "-"
+    character followed by a DIGIT, ignore the cookie-av.
```

Comment:
Should we accept negative numbers? That seems like surprising behavior (and I'm not sure it would meaningfully differ from just ignoring the cookie entirely since it would be expired when we constructed the next `cookie` header).

Reply:
Ok. 🤷  Fine by me if it matches our current behavior.

Reply:
+1 to accepting negative values.

Saying to ignore the cookie is somewhat ambiguous and could result in the cookie never getting set in the first place and therefore not overwriting any existing cookies in order to expire them.

E.x.:
`Set-Cookie: foo=bar; max-age=100000`
_A short time later_
`Set-Cookie: foo=delete_bar; max-age=-5`

I'd expect `foo` to be deleted after the second `Set-Cookie`